integration/framework/utils.py

```
@@ -151,7 +152,7 @@ def get_machine_info():
         return system
 
 
-def get_memory_info():
+def get_memory_info() -> str:
```

same here, the except pass is a really bad code smell